src/sw/redis++/redlock.hpp

```diff
@@ -0,0 +1,103 @@
+/**************************************************************************
+   Copyright (c) 2019
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+	   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ *************************************************************************/
+
+#ifndef REDISPLUSPLUS_REDLOCK_H
+#define REDISPLUSPLUS_REDLOCK_H
+
+#include "redis++.h"
+#include <unistd.h>
+#include <openssl/rc4.h>
```

Comment:
Since you don't use openssl stuff for the implementation of Redlock, you should not include this header. Otherwise, if the file is missing, i.e. openssl is not installed, your code won't compile.

Reply:
Thanks. This was a silly mistake from my side. The `openssl.h` include is now only in the test cases. See PR #33.